data/api/application-service/query_room.yaml

```diff
@@ -12,79 +12,99 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-swagger: '2.0'
+openapi: 3.1.0
 info:
-  title: "Matrix Application Service API"
-  version: "1.0.0"
-host: localhost:8008
-schemes:
-  - https
-  - http
-basePath: /_matrix/app/v1
-consumes:
-  - application/json
-produces:
-  - application/json
-securityDefinitions:
-  $ref: definitions/security.yaml
+  title: Matrix Application Service API
+  version: 1.0.0
 paths:
   "/rooms/{roomAlias}":
     get:
       summary: Query if a room alias should exist on the application service.
-      description: |-
-        This endpoint is invoked by the homeserver on an application service to query
+      description: >-
+        This endpoint is invoked by the homeserver on an application service to
+        query
+
         the existence of a given room alias. The homeserver will only query room
+
         aliases inside the application service's `aliases` namespace. The
+
         homeserver will send this request when it receives a request to join a
+
         room alias within the application service's namespace.
```

Comment:
The converter switched to the "folded" style which means newlines are only added to text when there's an empty line, which is similar to Markdown. However it expected the description to be in "literal" style before conversion, which means all newlines are kept, so it added empty lines to keep all the newlines from before, which is wrong, imo.

It should be noted that this doesn't have any effect on the rendering of the spec or in RapiDoc.

Reply:
yeah, we should clean these up.

I think the problem is that the definitions were using literal style to start with. If we went through and replaced `|-` with `>-` before doing the conversion, would that solve this problem?

Reply:
Sure, I'll test that

Reply:
Actually we have to keep the literal style, otherwise we lose new paragraphs with Hugo's markdownify (or we need to add an extra newline to get the new paragraph).

However, it should be possible to control the output serialization to make it use the same format.